Jenkinsfile.deploy

```
@@ -89,6 +131,14 @@ pipeline {
                 }
             }
         }
+        stage('Commit Changes'){
+            steps {
+                script {
+                    def COMMIT_MSG = "Update project version to ${params.PROJECT_VERSION}"
+                    githubscm.commitChanges(COMMIT_MSG)    
```

No need to create a variable here

```suggestion
                    githubscm.commitChanges("Update project version to ${params.PROJECT_VERSION}")    
```